newIDE/app/src/MainFrame/ElectronMainMenu.js

```diff
@@ -129,6 +130,10 @@ class ElectronMainMenu extends React.Component<Props, {||}> {
       ('update-status': MainMenuEvent),
       (event, status) => this._editor && this._editor.setUpdateStatus(status)
     );
+    ipcRenderer.on(
+      ('quit-app' : MainMenuEvent),
+      event => this._editor.quitApp()
```

Comment:
nit: no need to pass event if you are not going to use it

Reply:
actually you already have access to electron in this file. You can probably quit the app right here, but it would probably be better to ask for confirmation first

Reply:
I will be checking out right now.